pearreviw.txt

```diff
@@ -0,0 +1 @@
+just testing
```

Comment:
looks good

Reply:
Let me know if you are ready to merge.

Reply:
still waiting for your reply. Please confirm

Reply:
yes we can merge